docs/roadmap/releases.md

```diff
@@ -0,0 +1,101 @@
+necessary to release a product such as acs-engine, which depends on several external projects.
+
+"Components" applies not just to ACS projects, but also to development and release
+tools, orchestrator versions (Kubernetes, DC/OS, Swarm),to Docker base images, and to other Azure
+projects that do [semantic version][] releases.
+
+## acs-engine Releases Each Month
+
+acs-engine has a regular, public release cadence. From v0.1.0 onward, new acs-engine feature
+releases arrive on the first Thursday of each month. Patch releases are created at any time,
+as needed. GitHub milestones are used to communicate the content and timing of major and minor
+releases, and longer-term planning is visible at [the Roadmap](roadmap.md).
+
+acs-engine release timing is not linked to specific features. If a feature is merged before the
+release date, it is included in the next release.
+
+See "[How to Release acs-engine](#how-to-release-acs-engine)" for more detail.
+
+## Semantic Versioning
+
+acs-engine releases comply with [semantic versioning][semantic version], with the "public API" broadly
+defined as:
+
+- REST, gRPC, or other API that is network-accessible
+- Library or framework API intended for public use
+- "Pluggable" socket-level protocols users can redirect
+- CLI commands and output formats
+- Integration with Azure public APIs such as ARM
+
+In general, changes to anything a user might reasonably link to, customize, or integrate with should
+be backward-compatible, or else require a major release. acs-engine users can be confident that upgrading
+to a patch or to a minor release will not break anything.
+
+## How to Release acs-engine
+
+This section leads a maintainer through creating an acs-engine release.
+
+### Step 1: Assemble Master Changelog
+A change log is a file which contains a curated, chronologically ordered list of changes
+for each version of acs-engine, which helps users and contributors see what notable changes
+have been made between each version of the project.
+
+The CHANGELOG should be driven by release milestones defined on Github, which track specific deliverables and
+work-in-progress.
+
+### Step 2: Manual Testing
+
+Now it's time to go above and beyond current CI tests. Create a testing matrix spreadsheet (copying
+from the previous document is a good start) and sign up testers to cover all permutations.
+
+Testers should pay special attention to the overall user experience, make sure upgrading from
+earlier versions is smooth, and cover various storage configurations and Kubernetes versions and
+infrastructure providers.
+
+When showstopper-level bugs are found, the process is as follows:
+
+1. Create an issue that describes the bug.
+1. Create an PR that fixes the bug.
+1. Once the PR passes and is reviewed, merge it and update the CHANGELOG
+
+
+### Step 3: Tag and Create a Release
+
+TBD
+
+
+### Step 4: Close GitHub Milestones
+
+TBD
+
```

Comment:
We have used our "seed-repo" job in the past to close milestones and keep the labels in sync with each other.